### PR TITLE
[CLOSES #179] Plugin Publish and Revert to Groovy

### LINKED ISSUE #179: Revert Kotlin DSL Gradle Build to Groovy
- Restores artifact uploads and can be brought back later when artifact uploads are more robust in Kotlin DSL

### PR DESCRIPTION
**Pull request by adrw**
* [RELEASE] `misk-web-plugin:201902211512-8d6fdea`
* Publish to Gradle M2 repository
* Using https://plugins.gradle.org/docs/publish-plugin#kotlin-example
* Revert `misk-web` directory to Groovy from Kotlin DSL so internal `hooks.gradle` work
* Add to RELEASING.md instructions to publish plugin

### CODE REVIEW

**Comment by wesleyk** [comment]
The bullet points you listed out would have been _great_ granular commits. Can we try to do that in the future?

File: build.gradle.kts [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
 
-  val test by registering(GradleBuild::class) {
+  val testm2 by registering(GradleBuild::class) {
     dir = file("examples/gradle")
-    tasks = listOf("test")
+    tasks = listOf("testm2")
   }
 
-  test {
+  testm2 {
```
Comment (wesleyk) [inline build.gradle.kts]: why rename?

File: misk-web/settings.gradle [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//pluginManagement {
```
Comment (wesleyk) [inline misk-web/settings.gradle]: what's up with all this commented out stuff? Remove it or leave a note why it's commented out?

Comment (adrw) [reply misk-web/settings.gradle]: Oops I thought I had gotten all of it, I'll remove it and force push.

File: build.gradle.kts (hunk 1 above)
Comment (adrw) [reply build.gradle.kts]: To make it more clear that this is integration testing of the public published on M2 plugin vs the locally built version tested with `testlocal`.

Comment (wesleyk) [reply build.gradle.kts]: hmm okay, can you leave a comment on each task?

**Pull request merged**